content/guides/tree-shaking.md

```diff
@@ -84,3 +84,5 @@ function(e,t,n){"use strict";function r(e){return e*e*e}t.a=r}
 /* ... */
 function(e,t,n){"use strict";Object.defineProperty(t,"__esModule",{value:!0});var r=n(0);console.log(n.i(r.a)(5))}
 ```
+
+T> Note, using the `--optimize-minimize` flag is NOT required for tree shaking to occur. Tree shaking will occur when using a production configuration. This can also be configured directly in a `webpack.config.js` file. Read [production build](/guides/production) for an in-depth guide on how to create a production config file.
```

Comment:
Okay, I think that makes sense. However, I am unsure about the first part. Do you mean using something like [babel-plugin-resolver](https://github.com/tleunen/babel-plugin-module-resolver) when you say resolving using transpilers? I am not sure how transpilers work, but I am assuming loading your JS files with `babel-loader` does not resolve any imports, right?

What do you think about something along the lines of

> Note, using the `--optimize-minimize` flag is NOT required for tree shaking to occur. Tree shaking will occur when Webpack resolves your import/export statements and when you are including the `UglifyJsPlugin`. You can either include the `UglifyJsPlugin` using the `--optimize-minimize` flag or by adding the plugin manually to your `plugins` key of your configuration file. Read [production build](/guides/production) for an in-depth guide on how to create a production config file.